components/pluralrules/tests/operands.rs

```diff
@@ -33,3 +34,223 @@ fn test_parsing_operand_errors() {
     let operands: Result<PluralOperands, _> = "".parse();
     assert!(operands.is_err());
 }
+
+#[test]
+fn test_operand_eq() {
+    #[derive(Debug)]
+    struct TestCase {
+        o1: PluralOperands,
+        o2: PluralOperands,
+        expected: bool,
+    };
+    let tests = vec![
+        TestCase {
```

Comment:
Would we want to have it in data driven JSON tests? It feels like exactly the kind of behavior we'd want to test end-to-end between implementations.

Reply:
Done.